src/observers/buildStatusBarObserver.ts

```diff
@@ -9,7 +9,7 @@ export class BuildStatusBarObserver extends BaseStatusBarObserver {
                 this.setAndShowStatusBar(`$(sync~spin)`, undefined, undefined, 'Building the current workspace folder');
                 break;
             case EventType.BuildInstantReleased:
-                this.setAndShowStatusBar(`$(sync)`, undefined, undefined, 'Ready to Build');
+                this.setAndShowStatusBar(`$(sync)`, undefined, undefined, 'Ready to build');

```

Comment:
>build [](start = 84, length = 5)

validate

Reply:
I am kind of struggling between `build` and `validate` - would you agree at this moment `validate` is a better choice across the board?

Reply:
Yes, confirmed with Curt before, he also agree to use `validate`

---
In reply to: [393566233](https://github.com/docascode/vscode-docs-build/pull/98#discussion_r393566233) [](ancestors = 393566233)